05-services.md

```diff
@@ -1249,6 +1249,30 @@ There is a performance penalty for applications that swap memory to disk often.
 - If `memswap_limit` is unset, and `memory` is set, the container can use as much swap as the `memory` setting, if the host container has swap memory configured. For instance, if `memory`="300m" and `memswap_limit` is not set, the container can use 600m in total of memory and swap.
 - If `memswap_limit` is explicitly set to -1, the container is allowed to use unlimited swap, up to the amount available on the host system.
 
+## models
+
+`models` defines the models that the service containers use at runtime, referencing entries under the
+[top-level `models` key](models.md)
+
+```yaml
+services:
+  short_syntax:
+    image: app
+    models:
+      - my_model
+
+  long_syntax:
+    image: app
+    models:
+      my_model:
+        variable: MY_MODEL
+
+```
+
+When using the long syntax, `variable` can be set to define the environment variable set in container for model runner URL. 
```

Comment:
I think we confirmed offline, but `endpoint_var` might be a more descriptive name.